.travis.yml

```diff
@@ -41,24 +41,24 @@ jobs:
       stage: pre-test
       script: ./scripts/ci/ci_run_all_static_checks.sh
       env: >-
-        PYTHON_VERSION=3.6
+        PYTHON_MAJOR_MINOR_VERSION=3.6
```

Comment:
Why such strange name?

Reply:
It's not strange. It's MAJOR+MINOR version. PYTHON_VERSION is by default set in this image to 3.6.9 (so full PYTHON_VERSION) and what we really need here just MAJOR/MINOR.

Reply:
Are you OK with it @turbaszek ?